etc/purls.txt

```diff
@@ -6,7 +6,7 @@
 # re-generate the SBOM JSON file!
```

Comment:
Regenerate the SBOM file with the new dependency.

https://github.com/mongodb/mongo-cxx-driver/blob/b78767df7defed1320e0d0c0f159871a950dc1af/etc/releasing.md#check-and-update-the-sbom-lite includes instructions.

Secrets needed to follow SBOM steps are available under `drivers/cxx-driver` in [AWS Secrets Manager](https://wiki.corp.mongodb.com/display/DRIVERS/Using+AWS+Secrets+Manager+to+Store+Testing+Secrets).

Reply:
Bump: SBOM Lite must be regenerated with new purls.